src/marqo/s2_inference/reranking/rerank.py

```diff
@@ -1,15 +1,16 @@
 # use this as the entry point for reranking
-from marqo.s2_inference.reranking.enums import ResultsFields
-from marqo.s2_inference.reranking.cross_encoders import ReRankerText, ReRankerOwl
-from marqo.s2_inference.types import Dict, List
-from marqo.s2_inference.errors import RerankerError, RerankerNameError
 from PIL import UnidentifiedImageError
+
+from marqo.s2_inference.errors import RerankerError, RerankerNameError
 from marqo.s2_inference.logger import get_logger
+from marqo.s2_inference.reranking.cross_encoders import ReRankerText, ReRankerOwl
+from marqo.s2_inference.reranking.enums import ResultsFields
+from marqo.s2_inference.types import Dict, List
 
 logger = get_logger(__name__)
 
 def rerank_search_results(search_result: Dict, query: str, model_name: str, device: str, 
-                searchable_attributes: List[str] = None, num_highlights: int = 1, 
+                searchable_attributes: List[str], num_highlights: int = 1,
```

Comment:
Why did you remove the default value here? A bit concerned about changes here as I don't believe we have test coverage?

Reply:
There is a value check before called this function. But I can add it back.